Create ToDoItem

skipGit opened the pull request:
1. Create ToDoItem components.
2. Added some styles using makeStyles.

skipGit committed:
Create ToDoItem

1. Create ToDoItem components.
2. Added some styles using makeStyles.

DmitryKirilenko16 commented on package.json:
```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@emotion/styled": "^11.6.0",
```
Why do you need that lib?

DmitryKirilenko16 commented on src/components/ToDoItem.jsx:
```diff
@@ -0,0 +1,38 @@
+import { useDispatch, useSelector } from "react-redux";
+import DeleteSharpIcon from "@mui/icons-material/DeleteSharp";
+import CheckIcon from "@mui/icons-material/Check";
+import { useStyles } from "./to-do-item.style";
```
rename it to 'useToDoItemClasses'

DmitryKirilenko16 commented on src/components/ToDoItem.jsx:
```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import {
+  Card,
+  Typography,
+  Container,
+  IconButton,
+  CardContent,
+} from "@mui/material";
```
import components like (default import): import Card from "@mui/material/Card"

DmitryKirilenko16 commented on src/components/to-do-item.style.js:
```diff
@@ -0,0 +1,16 @@
+import { makeStyles } from "@mui/styles";
+
+export const useStyles = makeStyles({
```
rename it as comment above

skipGit committed:
Create ToDoItem fixes.

1. Remade material ui imports in default maner to resolve import conflicts.
2. Change components file tree for more convenient.
3. Removed unnecessary '@emotion/styled' lib.
